apps/portals/elportal/src/pages/HomePageV2.tsx

```diff
@@ -225,7 +245,28 @@ export default function HomePage() {
           style: { maxWidth: '100%' },
         }}
       >
-        <ELContributeYourData />
+        <SectionLayout ContainerProps={{ className: 'home-spacer' }}>
+          <ELContributeYourData />
+        </SectionLayout>
+        <SectionLayout ContainerProps={{ className: 'home-spacer' }}>
+          <PortalFeatureHighlights
+            image={analyzetheclouds}
+            title="Analyze on the Cloud"
+            buttonText="Learn About Our Analysis Platforms"
+            summaryText={styledPortalFeatureHighlightsSummaryText}
+            link="Analysis Platforms"
+          />
+        </SectionLayout>
+        <SectionLayout ContainerProps={{ className: 'home-spacer' }}>
+          <PortalFeatureHighlights
+            image={computationaltools}
+            title="Computational Tools"
+            buttonText="Check out our Computational Tools"
+            summaryText="We provide researchers with unique access to toolkits for probing and interpreting longevity-related data. The power of new software, pipelines, and applications are all at your fingertips. "
+            reverseOrder={true}
+            link="Explore/Computational Tools"
```

Comment:
HomePageV2 not at the root, please update relative paths to absolute ("/Explore/Computational Tools" for example)